src/components/CollectionTag.tsx

```diff
@@ -45,18 +46,20 @@ const PropertyLabel = styled(Label)`
 const DensityLabel = styled(Label)`
 `;
 
-const CollectionTag: SFC<Props> = ({ isOpen, index, toggleOpen, collection, currentCollectionListId, dataSetId }) => {
-    const { title, collectionId, collectionListId, properties } = collection;
+const renderButtonType = (CollectionIsKnown: boolean, CollectionIsSelected: boolean) => {
+    if (!CollectionIsKnown) {
+        return BUTTON_TYPES.disabled;
+    }
 
-    let buttonType = currentCollectionListId && currentCollectionListId === collectionListId
+    return CollectionIsSelected
         ? BUTTON_TYPES.dark
         : BUTTON_TYPES.inverted;
-    
-    let tagTitle = collectionId;
-    if (collectionId.indexOf('vocabulary_unknown') !== -1) {
-        tagTitle = 'Uknown';
-        buttonType = BUTTON_TYPES.disabled;
-    }
+};
+
+const UNKNOWN: string = Translations.translate('details.collection.unknown');
```

Comment:
Naast dat het (stylistic) onnodig is om deze vertaling naar een apart variabel te verplaatsen ipv direct translate aan te roepen, is dit waarschijnlijk een bug; als de taal verandert, verandert de inhoud van deze string niet